app/controllers/concerns/foreman/controller/parameters/discovery_rule.rb

```diff
@@ -5,6 +5,8 @@ module Foreman::Controller::Parameters::DiscoveryRule
     def discovery_rule_params_filter
       Foreman::ParameterFilter.new(::DiscoveryRule).tap do |filter|
         filter.permit :name, :search, :hostname, :priority, :enabled, :hostgroup, :hostgroup_id, :max_count
+        filter.permit :organization_ids => []
```

Comment:
Thanks @domcleal amended, it now includes unit tests around these params. RTM @orrabin